packages/iTwinUI-react/src/core/Select/Select.tsx

```diff
@@ -55,19 +89,42 @@ export type SelectOption<T> = {
   [key: string]: unknown;
 } & CommonProps;
 
+export type SelectValueChangeEvent = 'Added' | 'Removed';
```

Comment:
any reason to go with `Added` instead of `added`?

Reply:
changed